.ci/docker/conan-tests

```diff
@@ -20,8 +20,9 @@ ENV PY36=3.6.15 \
     GXX_11=/usr/bin/g++-11 \
     CLANG_14=/usr/bin/clang-14 \
     CLANGXX_14=/usr/bin/clang++-14 \
-    BAZEL_6_3_2=6.3.2 \
-    BAZEL_7_1_2=7.1.2
+    BAZEL_6_5_0=6.5.0 \
+    BAZEL_7_4_1=7.4.1 \
+    BAZEL_8_0_0=8.0.0
```

Comment:
Minor suggestion, name this with the major so we just need to change this part for future updates:

```suggestion
    BAZEL_6=6.5.0 \
    BAZEL_7=7.4.1 \
    BAZEL_8=8.0.0
```